package.json

```diff
@@ -1,21 +1,21 @@
 {
   "dependencies": {
-    "@azure-tools/typespec-autorest": "^0.48.0",
-    "@azure-tools/typespec-azure-core": "^0.48.0",
-    "@azure-tools/typespec-azure-resource-manager": "^0.48.0",
-    "@azure-tools/typespec-client-generator-core": "^0.48.4",
-    "@typespec/compiler": "^0.62.0",
-    "@typespec/http": "^0.62.0",
+    "@azure-tools/typespec-autorest": "^0.49.0",
+    "@azure-tools/typespec-azure-core": "^0.49.0",
+    "@azure-tools/typespec-azure-resource-manager": "^0.49.0",
+    "@azure-tools/typespec-client-generator-core": "^0.49.0",
+    "@typespec/compiler": "^0.63.0",
+    "@typespec/http": "^0.63.0",
     "@typespec/http-client-csharp": "^0.1.9-alpha.20241210.2",
     "@typespec/http-server-csharp": "^0.58.0-alpha.5",
     "@typespec/http-server-javascript": "file:./pkg/typespec-http-server-javascript-0.58.0-alpha.5.tgz",
     "@typespec/http-client-java": "^0.1.3",
-    "@azure-tools/typespec-ts": "^0.35.0",
+    "@azure-tools/typespec-ts": "^0.36.0",
```

Comment:
@ArcturusZhang we need to upgrade compiler to v0.63 for todo spec generation and which caused dependency version conflict becasue other emitters are still in v0.62, currently we temporarily disabled this error with `npm install --force`. Not sure this would block our pr merging.

Reply:
I would hold on this pr until all emitters upgrade to v0.63 otherwise we need to force install every time.